plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -132,7 +132,8 @@ void chain_api_plugin::plugin_startup() {
       CHAIN_RW_CALL_ASYNC(push_block, chain_apis::read_write::push_block_results, 202, http_params_types::params_required),
       CHAIN_RW_CALL_ASYNC(push_transaction, chain_apis::read_write::push_transaction_results, 202, http_params_types::params_required),
       CHAIN_RW_CALL_ASYNC(push_transactions, chain_apis::read_write::push_transactions_results, 202, http_params_types::params_required),
-      CHAIN_RW_CALL_ASYNC(send_transaction, chain_apis::read_write::send_transaction_results, 202, http_params_types::params_required)
+      CHAIN_RW_CALL_ASYNC(send_transaction, chain_apis::read_write::send_transaction_results, 202, http_params_types::params_required),
+      CHAIN_RO_CALL_WITH_400(get_all_accounts, 200, http_params_types::params_required)
```

Comment:
Use `CHAIN_RO_CALL` instead. Not sure why we have both `CHAIN_RO_CALL` and `CHAIN_RO_CALL_WITH_400` they look identical.

Reply:
Done